Client/Frontend/Browser/Tab.swift

```diff
@@ -385,7 +385,7 @@ class Tab: NSObject {
                 URL: displayURL,
                 title: tab.displayTitle,
                 history: history,
-                lastUsed: Date.now(),
+                lastUsed: tab.lastExecutedTime ?? 0,
```

Comment:
Not sure what happened here, it seems I fixed this but didn't properly commit it earlier 🙈

Reply:
@OrlaM Just out of curiosity, do you know under which circumstances `lastExecutedTime` would be nil?

Reply:
I can't immediately see any. I have jotted this down as something to look into as part of the refactor. This value really shouldn't be an optional.

Reply:
> I can't immediately see any. I have jotted this down as something to look into as part of the refactor. This value really shouldn't be an optional.

Sounds good--thanks!